pydatajson/core.py

```diff
@@ -737,6 +737,55 @@ def _extract_datasets_to_harvest(cls, report):
 
         return datasets_to_harvest
 
+    def generate_catalog_indicators(self, catalogs):
+        """Genera una lista de diccionarios con varios indicadores sobre
+        los catálogos provistos, tales como la cantidad de datasets válidos,
+        días desde su última fecha actualizada, entre otros.
+
+        Args:
+            catalogs (str o list): uno o más catalogos sobre los que se quiera
+            obtener indicadores
```

Comment:
En los docstring las líneas de descripción de cada argumento que se pasen de la primera van indentadas para que se lean bien dps en la doc automática en Read The Docs.
```
        catalogs (str o list): uno o más catalogos sobre los que se quiera
        obtener indicadores
```
pasa a ser
```
        catalogs (str o list): uno o más catalogos sobre los que se quiera
            obtener indicadores
```